Formula/cql-proxy.rb

```diff
@@ -0,0 +1,19 @@
+class CqlProxy < Formula
+  desc "DataStax cql-proxy enables Cassandra apps to use Astra DB without code changes"
+  homepage "https://github.com/datastax/cql-proxy"
+  url "https://github.com/datastax/cql-proxy/archive/refs/tags/v0.1.1.tar.gz"
+  sha256 "25cd9a87078c11caed7da14d2b8b85e5240bfe1aff5b666d0dcad9e73c15b305"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    exec("touch secure.txt")
```

Comment:
```suggestion
    touch "secure.txt"
```

Reply:
@carlocab  Not sure what the difference between these two, but without the `exec` the test fails.  Maybe it writes it into another directory?

Reply:
So this PR has been out here for a couple of months.  Is the `exec` difference worth getting to the bottom of?

Reply:
The reason `exec("touch secure.txt")` works is because this skips the subsequent `shell_output` and `assert_match`. When using `exec`, the existing process is replaced with the one invoked through `exec`. Basically, the test process is `exec`ing `touch secure.txt` and immediately exiting.

The error when using `touch "secure.txt"` appears to be that the exit code of `cql-proxy` is 1 due to the `unable to open` error. I would expect that this is the intended behavior. By default, `shell_output` will expect an exit code of 0.

`shell_output` accepts the expected exit code an optional second argument. So this should work with `touch "secure.txt"`:
```ruby
    assert_match "unable to open", shell_output("cql-proxy -b secure.txt --bind 127.0.0.1 2>&1", 1)
```